go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/anchore/stereoscope
 
-go 1.22.9
+go 1.24.0
+
+toolchain go1.24.1
```

Comment:
FWIW I would rather avoid getting these `toolchain` entries, if possible. I thought that would happen with a third segment in the `go` directive like `go 1.24.0` 🤷

Reply:
I'm not sure what I should do differently to avoid leaving that there - manually deleting it makes me feel like I'm missing something. I'll keep looking, but if you have steps for how to avoid this, please feel free to share :)

Reply:
@kzantow making the `go` statement have `1.24.1` made the `toolchain` line go away.